tests/integration/test_alerts.py

```diff
@@ -1,23 +1,25 @@
 from datetime import datetime
 from datetime import timedelta
+from shlex import split as split_command
```

Comment:
Used `shlex` to parse string `cmd a --b c`  into a `list: [ "cmd", "a", "--b", "c"].  
Specifying the command in a string is easier to read and write.

Reply:
Why not just do `.split(" ")` ? Why do we need a whole new depedency to do this?

Reply:
when inputs like "2020-01-02 00:00:00" or ' New case' it will parse incorrectly with split(" "),

Reply:
Looks like it's a standard library anyway, so it's not "another dependency" as I had thought